### PR TITLE
Feature/aw/filter header

### PR DESCRIPTION
**Pull request by 6Botter7**
Review Required

### CODE REVIEW

File: components/filter-header/filter-header.html [hunk 1]
```diff
@@ -38,21 +38,12 @@
         cursor: pointer;
     }
 
-    #btnClose {
-        display: var(--close-display);
+    .icon{
+        font-family: crsfrw;
+        font-size: 1.5rem;
```
Comment (Ashton-Pragma) [inline components/filter-header/filter-header.html]: check with chuck

File: components/filter-header/filter-header.js [hunk 2]
```diff
@@ -1,25 +1,81 @@
...
+
+    /**
+     * @method disconnectedCallback - When the component is removed from the DOM, the container is set to null
+     */
     async disconnectedCallback() {
         this.#container = null;
-        await super.disconnectedCallback();
+        this.shadowRoot.querySelector("input").removeEventListener("keyup", this.#filterHandler);
+        await crsbinding.translations.delete("filterHeader");
+        this.#filterHandler = null;
```
Comment (Ashton-Pragma) [inline components/filter-header/filter-header.js:65]: check the memory tool

**Review by Ashton-Pragma — CHANGES_REQUESTED** [review]

File: components/filter-header/filter-header.html (hunk 1 above)
Comment (6Botter7) [reply components/filter-header/filter-header.html]: Does not have to be class icon, as long as font-family: crsfrw is there.

File: components/filter-header/filter-header.js (hunk 2 above)
Comment (6Botter7) [reply components/filter-header/filter-header.js:65]: Checked with tool, no memory leaks were found

**Pull request closed without merging**